src/components/ItaliaTheme/View/NewsItemView/NewsItemText.jsx

```diff
@@ -13,14 +16,23 @@ const messages = defineMessages({
 const NewsItemText = ({ content }) => {
   const intl = useIntl();
 
-  return (
+  // NewsItem ha la descrizione_estesa (testo con blocchi)
+  // altri CT (come i Comunicati Stampa) potrebbero avere `text`
+  // (richtext senza blocchi).
+  //
+  // TODO: al momento viene presentato uno solo dei due, valutare se
+  // mettere entrambi, il componente comuqneu non mostra nulla se il
+  // contenuto è vuoto.
+  return content.descrizione_estesa ? (
```

Comment:
Questa condizione potrebbe non essere garantita se scritta così. Vedi per esempio come è fatto in `src/components/ItaliaTheme/View/DocumentoView/DocumentoDescrizione.jsx` usando `richTextHasContent`

Reply:
concordo

Reply:
in questo caso a me non interessa il fatto che sia vuoto, mi interessa più il fatto che esiste il campo, se il campo esiste nel contenttype allora è quello che ha priorità ed giusto da mostrare (eventualmente anche vuoto mi sta bene)

Reply:
ho aggiunto un commento che spiega la cosa

Reply:
la condizione sul fatto che sia compilato o meno va messa, perchè altrimenti la sezione viene mostrata senza il coontenuto, e poi compare nel menu laterale anche se vuota

Reply:
vero quanto dice giulia. capisco meglio ora però la logica del componente che hai scritto. Io proporrei così:

```jsx
return (
  <>
    {content.descrizione_estesa && richTextHasContent(content.descrizione_estesa) && (
      <RichTextSection ... />
    )}
    {!content.descrizione_estesa && <RichText data={content.text} />}
  </>
);
```

Ti torna così?

Reply:
no. deve mostrare il menu se la proprietà esiste, anche se è vuota. 
Ho cambiato la condizione per rendere più chiaro e esplicito il concetto.

Reply:
non è detto che quello che c'era prima era corretto. Dato che ce ne siamo accorti che la condizione non è giusta facciamolo bene. 
in ogni caso, quello che proporrei io è 

```
return (
  <>
    {content.descrizione_estesa && richTextHasContent(content.descrizione_estesa) && (
      <RichTextSection
        data={content.descrizione_estesa}
        tag_id={'text-body'}
        field="descrizione_estesa"
        title={intl.formatMessage(messages.news_item_contenuto)}
        show_title={false}
      />
    )}
    {!content.descrizione_estesa && 
      <RichTextSection     
            tag_id={'text-body'}
            title={intl.formatMessage(messages.news_item_contenuto)}
            show_title={false}
          >
                <RichText data={content.text} />
         </RichTextSection>
     }
  </>
);
```